client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.ts

```diff
@@ -33,6 +33,7 @@ import { InteractionService } from 'src/app/site/pages/meetings/pages/interactio
 import { SortingListComponent } from 'src/app/ui/modules/sorting/modules/sorting-list/components/sorting-list/sorting-list.component';
 import { PromptService } from 'src/app/ui/modules/prompt-dialog';
 import { PointOfOrderDialogService } from '../../modules/point-of-order-dialog/services/point-of-order-dialog.service';
+import { UserSelectionData } from '../../../user-create-search-selector/components/user-create-search-selector/user-create-search-selector.component';
```

Comment:
Please export exported data from ui modules from the root of an ui module. Then, it can be imported like so:

```suggestion
import { UserSelectionData } from `ui/modules/user-create-search-selector`
```